.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: "Experimental tests"
         run: ./build/linux/amd64/earth -P ./examples/tests+experimental
       - name: Execute test similar to homebrew test in https://github.com/Homebrew/homebrew-core/blob/master/Formula/earthly.rb
-        run: ./build/linux/amd64/earth --buildkit-host 127.0.0.1 ./examples/tests/with-docker+all | grep 'Error while dialing invalid address 127.0.0.1'
+        run: ./build/linux/amd64/earth --buildkit-host 127.0.0.1 ./examples/tests/with-docker+all 2>&1 | grep 'Error while dialing invalid address 127.0.0.1'
```

Comment:
luckily homebrew already combines stderr and stdout in https://github.com/homebrew/homebrew-core/blob/master/Formula/earthly.rb#L41